src/Components/Mission.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { JoinMission } from '../Redux/missions/Mission';
+
+const Mission = (props) => {
+  const { missionInfo } = props;
+  const dispatch = useDispatch();
+
+  /* const Join = () => {
+    dispatch(JoinMission(missionInfo.id));
+  }; */
```

Comment:
- [ ] Kindly remove this commented code. It makes your codebase look messy, and hard to read.